server.js

```diff
@@ -2,32 +2,41 @@ import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
+import booksData from './data/books.json'
+
 const port = process.env.PORT || 8080
 const app = express()
 
-// Add middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Welcome to Amanda\'s Book API!')
+})
+
+app.get('/books', (req, res) => {
+  res.json(booksData)
+})
+
+app.get('/books/:id', (req, res) => {
+  const id = req.params.id
+  const book = booksData.find(item => {
+    return (item.bookID === +id)
+  })
+  if (book) {
+    return res.json(book)
+  } else {
+    return res.status(404).json({ error: 'Cannot find the book you are looking for' })
+  }
+})
+
+app.get('/author/:author', (req, res) => {
+  const author = req.params.author
+  const booksByAuthor = booksData.filter((item) => item.authors.toLowerCase().includes(author))
+  res.json(booksByAuthor)
 })
```

Comment:
Nice with some filtering! Since this endpoint too returns books, I think you could have this as a query param under your `/books` endpoint, like:
`/books?author=rowling`
⭐  for `toLowerCase()` and `includes()`